### PR TITLE
Seed node repo refactoring

### PR DESCRIPTION
**Pull request by freimair**
github.com/AlDanial/cloc v 1.80  T=0.07 s (14.0 files/s, 14.0 lines/s)

_ | files | blank | comment | code
:-------|-------:|-------:|-------:|-------:
 same|0|0|98|645
 modified|5|0|0|20
 added|1|18|22|57
 removed|7|97|145|321

### CODE REVIEW

File: core/src/test/resources/testnet.seednodes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# nodeaddress.onion:port [(@owner)]
```
Comment (ManfredKarrer) [inline core/src/test/resources/testnet.seednodes:1]: They are all removed. Better not have them in the code otherwise people think they are running.

**Review by ManfredKarrer — CHANGES_REQUESTED** [review]

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Comment by ManfredKarrer** [comment]
@freimair I reverted that PR as I had issues with seednodes when using it. Can you please have a look and open a new PR with that content and tested with existing seed node setup (usual prog args).

**Comment by freimair** [comment]
https://github.com/bisq-network/bisq/commit/a9ee15d9f60b52d0c5e2e3a9cc24e9be82a8b407#commitcomment-32515049

**Comment by freimair** [comment]
nevermind: https://github.com/bisq-network/bisq/pull/2476

shame on me

**Pull request merged**